WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListViewModel.kt

```diff
@@ -81,7 +80,20 @@ class NotificationsListViewModel @Inject constructor(
             }
     }
 
+    fun likeComment(note: Note, liked: Boolean) {
+        val site = siteStore.getSiteBySiteId(note.siteId.toLong()) ?: return
+        note.setLikedComment(liked)
+        _updatedNote.postValue(note)
+        viewModelScope.launch {
```

Comment:
I've noticed that on some occasions the `site` object is not retrieved because it is not available on the local database (I was testing with comments from https://github.com/wordpress-mobile/WordPress-Android/pull/20186).

I think we could try fetching it 

or fake it since only the siteId is used in the call 😅 
```
        val site =
            siteStore.getSiteBySiteId(note.siteId.toLong()) ?: SiteModel().apply {
                siteId = note.siteId.toLong()
                setIsWPCom(true)
            }
```

Reply:
I'd like to fake it first then make refactors 😅